### PR TITLE
Active Directory WP added & some changes

### PR DESCRIPTION
**Pull request by zprood**
## Done

[Active Directory WP takeover]
[Engage page created]
[Small changes]

PS: I found if I can update and create takeover&pages on my forked branch, then create the PR will be more efficiently.

### CODE REVIEW

**Comment by solazio** [comment]
@zprood there is an issue when trying to download the pdf. It looks like the link is broken.

**Comment by zprood** [comment]
> @zprood there is an issue when trying to download the pdf. It looks like the link is broken.
Hi solazio. I write the download link with absolute address, the download page is under /engage directory (thank_you_ad.html), when the PR is merged, the download link will be live too.

File: templates/engage/thank_you_ad.html [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+{% block title %}将Ubuntu桌面与Active Directory集成{% endblock %}
+
+{% block content %}
+
+<div class="p-strip">
+  <div class="row">
+    <div class="col-7">
+<h1 class="p-heading--two">谢谢！</h1>
+<h3>你的白皮书现已可下载...</h3>
+<p><a href="https://pages.ubuntu.com/rs/066-EOV-335/images/Canonical%20-%20Active%20Directory_7.5.19-CN.pdf" class="link-cta-ubuntu cta-large" target="download">立即下载</a></p>
```
Comment (solazio) [inline templates/engage/thank_you_ad.html]: please fix indentation.

File: templates/engage/microsoft-active-directory.html [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+        </li>
+        <li class="p-list__item is-ticked">
+          集成 Ubuntu 到 Active Directory的其他工具。
+        </li>
+      </ul>
+    </div>
+    <div class="col-5">
+
+    <!-- MARKETO FORM -->
+<script src="//assets.ubuntu.com/v1/37b1db88-jquery.min.js"></script>
```
Comment (solazio) [inline templates/engage/microsoft-active-directory.html]: you don't need https:// ... here?

File: templates/engage/microsoft-active-directory.html [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+    var selection = $(this).val();
+    if (selection == 'Commercially only') {
+      $('#comments').show();
+    } else {
+      $('#comments').hide();
+    }
+  });
+
+</script>
+<script src="//assets.ubuntu.com/v1/f97fa297-stateCountry.js"></script>
```
Comment (solazio) [inline templates/engage/microsoft-active-directory.html]: https://... ?

File: templates/engage/thank_you_ad.html (hunk 1 above)
Comment (zprood) [reply templates/engage/thank_you_ad.html]: Changed to col-8+col-4, is it ok ?

Comment (solazio) [reply templates/engage/thank_you_ad.html]: @zprood sorry, is not what I meant. Is just the code if you can indent it properly. Like h1, h3 & p to start 2 spaces to the right of the parent div.

Comment (zprood) [reply templates/engage/thank_you_ad.html]: @solazio aha, got your point. The tag's position changed.

**Comment by zprood** [comment]
@solazio Thanks !

**Pull request merged**